international-affairs.csl

```diff
@@ -0,0 +1,285 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" default-locale="en-GB">
+  <info>
+    <title>International Affairs</title>
+    <id>http://www.zotero.org/styles/international-affairs</id>
+    <link href="http://www.zotero.org/styles/international-affairs" rel="self"/>
+    <link href="http://www.zotero.org/styles/textual-practice" rel="template"/>
+    <link href="https://academic.oup.com/ia/pages/Author_Guidelines?login=false" rel="documentation"/>
+    <link href="https://static.primary.prod.gcms.the-infra.com/static/site/ia/document/Document-Library/House-Style-March-2023.pdf?node=ff7f174a9525c6192650" rel="documentation"/>
+    <author>
+      <name>Patrick O'Brien</name>
+    </author>
+    <category citation-format="note"/>
+    <category field="humanities"/>
+    <issn>0020-5850</issn>
+    <eissn>1468-2346</eissn>
+    <updated>2024-10-14T15:42:14+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
+    <terms>
+      <term name="editor" form="short">
+        <single>ed.</single>
+        <multiple>eds.</multiple>
+      </term>
+      <term name="translator" form="short">trans.</term>
+      <term name="editortranslator" form="short">ed. and trans.</term>
+    </terms>
+  </locale>
+  <macro name="edition">
+    <choose>
+      <if is-numeric="edition">
+        <group delimiter=" ">
+          <number variable="edition" form="ordinal"/>
+          <label variable="edition" form="short"/>
+        </group>
+      </if>
+      <else>
+        <text variable="edition"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="author">
```

Comment:
@POBrien333 are you sure this shouldn't have a `<substitute>` element? See the Beyond Varieties... sample citation, which looks weird.

Reply:
If we don't have an indication otherwise, let's include our standard editor -> title substitution

Reply:
Thanks for spotting. Sure.